lib/scrum/sprint_cleaner.rb

```diff
@@ -1,15 +1,16 @@
 module Scrum
   class SprintCleaner < TrelloService
-    SOURCE_LISTS = ["Sprint Backlog", "Doing"]
-    TARGET_LIST = "Ready"
+    TARGET_LIST = "Ready for Estimation"
```

Comment:
This means that anyone using trollolo should change the name of this list right? We should at least add a record in the CHANGELOG to warn everybody.